src/lib/serviceRoutines/entryPointsTreat.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+*
+* Copyright (c) 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: José Manuel Cantera
+*/
+#include <string>
+#include <vector>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
+#include "common/string.h"
+#include "common/globals.h"
+#include "common/tag.h"
+
+#include "ngsi/ParseData.h"
+#include "rest/ConnectionInfo.h"
+#include "serviceRoutines/entryPointsTreat.h"
+
+
+
+/* ****************************************************************************
+*
+* entryPointsTreat -
+*/
+std::string entryPointsTreat
+(
+  ConnectionInfo*            ciP,
+  int                        components,
+  std::vector<std::string>&  compV,
+  ParseData*                 parseDataP
+)
+{
+  std::string out     = "{";
+  std::string indent  = "";
+
+  out += valueTag(indent,        "entities_url",      ENTITIES_URL,      ciP->outFormat, true);
+  out += valueTag(indent + "  ", "types_url",         TYPES_URL,         ciP->outFormat, true);
+  out += valueTag(indent + "  ", "subscriptions_url", SUBSCRIPTIONS_URL, ciP->outFormat, true);
+  out += valueTag(indent + "  ", "registrations_url", REGISTRATIONS_URL, ciP->outFormat, false);
+
```

Comment:
The functions in tag.cpp/h are meant for v1 (where JSON and XML are supported).
For v2 there are no helper functions yet and I'm not sure there will be any.
There functions add indents and newlines (that we don't want for v2), so please reimplement these four lines without  valueTag:

    out += "\"entities_url\":\"" + ENTITIES_URL + "\",";  ...